tests/v2/validation/connection/workload.go

```diff
@@ -0,0 +1,23 @@
+package connection
+
+import (
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/extensions/workloads"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	DeploymentSteveType = "apps.deployment"
```

Comment:
We have a stevetypes constant file in shepherd (extensions/defaults/stevetypes/stevetypes.go). Please add this constant to the list so it can be reused across the framework.

Reply:
Hi @susesgartner I've took a look and we already have a constant in shepherd\extensions\workloads\workloads.go, I'll reuse the constant